cobalt/BUILD.gn

```diff
@@ -33,6 +33,7 @@ group("gn_all") {
   }
   if (is_cobalt_hermetic_build) {
     deps += [
+      "//cobalt:cobalt_loader",
```

Comment:
nit: change to just `:cobalt_loader` (it's defined in this file, though done implicitly via the template)